integration_tests/utils/Cases.jl

```diff
@@ -711,7 +709,7 @@ function initialize_profiles(self::CasesBase{Rico}, Gr::Grid, GMV::GridMeanVaria
 
     @inbounds for k in real_center_indices(Gr)
         GMV.H.values[k] = thetal[k]
-        ts = TD.PhaseEquil_pθq(param_set, Ref.p0_half[k], GMV.H.values[k], GMV.QT.values[k])
+        ts = TD.PhaseEquil_pθq_anelastic(param_set, Ref.p0[k], GMV.H.values[k], GMV.QT.values[k])
```

Comment:
```suggestion
        ts = TD.PhaseEquil_pθq_anelastic(param_set, Ref.p0_half[k], GMV.H.values[k], GMV.QT.values[k])
```
This happens in several places.